lib/wordpress.js

```diff
@@ -44,7 +44,9 @@ function Client( settings ) {
 		// via settings.host
 		headers: {
 			Host: parsedUrl.host
-		}
+		},
+
+		basic_auth: settings.basic_auth
```

Comment:
btw... there are some more "misnamed" options in the game. as it stands right now, even when using `basicAuth` as a key the config would look like

```
{
	"url": "https://example.com/",
	"username": "test123",
	"password": "xdsfadsf",
    "basicAuth": {
        "user": "testr5z57",
        "pass": "asdfadsf"
    }
}
```

so username vs. user
and password vs. pass